### PR TITLE
INN-3283: some vercel fixups

### PR DESCRIPTION
**Pull request by jacobheric**
## Description

* fix vercel project not found flash bug
* add reconnect button
* remove old vercel integration flow and feature flags

## Motivation
clean up

## Type of change (choose one)
- [ ] Chore (refactors, upgrades, etc.)
- [x] Bug fix (non-breaking change that fixes an issue)
- [ ] Security fix (non-breaking change that fixes a potential vulnerability)
- [ ] Docs
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality not to work as expected)

## Checklist
- [x] I've linked any associated issues to this PR.
- [x ] I've tested my own changes.

*[Check our Pull Request Guidelines](https://github.com/inngest/inngest/blob/main/docs/PULL_REQUEST_GUIDELINES.md)*

### CODE REVIEW

File: ui/apps/dashboard/src/app/(organization-active)/(dashboard)/settings/integrations/vercel/configure/[id]/page.tsx [hunk 1]
```diff
@@ -127,11 +128,33 @@ export default function VercelConfigure() {
   // when enabling a project (e.g. the protection bypass secret)
   const areExtraSettingsVisible = project?.isEnabled && newEnablement !== 'enabled';
 
+  if (fetching) {
```
Comment (jacobheric) [inline ui/apps/dashboard/src/app/(organization-active)/(dashboard)/settings/integrations/vercel/configure/[id]/page.tsx:131]: Handle all data fetching states explicitly and check for data and project below. This removes the erroneous flash.

File: ui/apps/dashboard/src/app/(organization-active)/(dashboard)/settings/integrations/vercel/page.tsx [hunk 2]
```diff
@@ -43,12 +38,11 @@ export default async function VercelIntegrationPage() {
...
           <NewButton
-            appearance="outlined"
-            kind="secondary"
-            href={'https://vercel.com/integrations/inngest' as Route}
-            icon={<RiExternalLinkLine className="mr-1" />}
+            kind="primary"
+            href={'https://vercel.com/integrations/inngest/new' as Route}
+            icon={<RiLinksLine className="mr-1" />}
             iconSide="left"
-            label="Go to Vercel"
```
Comment (jacobheric) [inline ui/apps/dashboard/src/app/(organization-active)/(dashboard)/settings/integrations/vercel/page.tsx:51]: Changed this button as well per latest proj-vercel-int-mgmt feedback.

File: ui/apps/dashboard/src/app/(organization-active)/(dashboard)/settings/integrations/vercel/configure/[id]/page.tsx [hunk 3]
```diff
@@ -127,11 +128,33 @@ export default function VercelConfigure() {
...
+
+  if (fetchError) {
+    return (
+      <div className="flex h-full w-full items-center justify-center">
+        <Alert severity="error">{fetchError.message}</Alert>
+      </div>
+    );
+  }
+
+  if (data.projects.length === 0) {
```
Comment (jacobheric) [inline ui/apps/dashboard/src/app/(organization-active)/(dashboard)/settings/integrations/vercel/configure/[id]/page.tsx:147]: This is the actual bugfix.

**Comment by goodoldneon** [comment]
This PR fixes the bug, but there's one more (possibly preexisting) thing: the page is blank when I have an invalid Vercel project ID in the URL

**Review by djfarrelly — APPROVED** [review]

**Comment by jacobheric** [comment]
> > This PR fixes the bug, but there's one more (possibly preexisting) thing: the page is blank when I have an invalid Vercel project ID in the URL
> 
> ok, will fix

fixed

**Pull request merged**